MANIFEST.in

```diff
@@ -10,6 +10,7 @@ include .pre-commit-config.yaml
 include long_description.rst
 
 recursive-exclude tools *
+recursive-exclude tests *
```

Comment:
As a downstream maintainer, would very much like to see `tests/` and any required configuration still included in the `sdist`.

Reply:
Opened https://github.com/ipython/ipython/pull/14746

Reply:
Yep, it was an oversight, and thanks for #14746